spec/features/admin/product_import_spec.rb

```diff
@@ -79,8 +79,10 @@
 
     it "displays info about invalid entries but no save button if all items are invalid" do
       csv_data = CSV.generate do |csv|
-        csv << ["name", "producer", "category", "on_hand", "price", "units", "unit_type"]
-        csv << ["Bad Carrots", "Unkown Enterprise", "Mouldy vegetables", "666", "3.20", "", "g"]
+        csv << ["name", "producer", "category", "on_hand", "price", "units", "unit_type", "shipping_category_id"]
```

Comment:
The importer doesn't generally use `shipping_category_id` as a field (although it can). It's usually `shipping_category`, and uses the name instead of id. Could you use `shipping_category.name` in place of `shipping_category_id_str` and change the column name from `"shipping_category_id"` to `"shipping_category"`?